discodos/views.py

```diff
@@ -502,6 +502,14 @@ def __init__(self):
             15: {"width": 100, "hidden": False},  # MusicBrainz Match T.
         }
 
+        self.column_defaults_treeview = {
```

Comment:
please put it to mix related class. between here is good: https://github.com/JOJ0/discodos/blob/d4489ff075e9fbc966deb190d7ee86cc55d4aca0/discodos/views.py#L374

Reply:
and rename to column_defaults_mixes